src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+//import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import styled from 'styled-components';
+
+import Searchbar from '../Searchbar';
+import DarkModeToggle from '../DarkModeToggle';
+import BurgerMenu from '../BurgerMenu';
+import LoginAvatar from '../LoginAvatar';
+
+const Header = styled.header`
+  background: #c4b5de;
+  padding: 0.25em 0.25em;
+  height: 5vh;
+  min-height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: flex;
+`;
+
+const MainHeader = () => {
```

Comment:
It is a good practice to name the function name component as the file name, maybe we can rename the file name to MainHeader.component.jsx or name our component function as Header, what do you think?

Reply:
I agree. I think I changed the name of the component function so as to prevent a clash with the styled component name but I have changed this to match the file name with the component function. Thanks 🙂